SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -4,6 +4,7 @@
 
 #include "CoreMinimal.h"
 #include "Engine/EngineTypes.h"
+#include "Interop/Connection/SpatialEventTracer.h"
```

Comment:
Hmm, I wonder if it is worth pulling the enum (EEventTraceFileOutputType) into a separate file? 
Our code includes SpatialGDKSettings.h everywhere :(

Reply:
Do we want to do this?

Reply:
I don't think the include is actually even required anymore (the enum was removed) - so will update!